src/SprykerEco/Zed/UnzerGui/Dependency/UnzerGuiToUnzerFacadeBridge.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Dependency;
+
+use Generated\Shared\Transfer\MerchantUnzerParticipantTransfer;
+
+class UnzerGuiToUnzerFacadeBridge implements UnzerGuiToUnzerFacadeInterface
+{
+    /**
+     * @var \SprykerEco\Zed\Unzer\Business\UnzerFacadeInterface
+     */
+    protected $unzerFacade;
+
+    /**
+     * @param \SprykerEco\Zed\Unzer\Business\UnzerFacadeInterface $unzerFacade
+     */
+    public function __construct($unzerFacade)
```

Comment:
```suggestion
    public function __construct(UnzerFacadeInterface  $unzerFacade)
```

Missing type hint?

Reply:
No, it is a bridge. https://spryker.atlassian.net/wiki/spaces/RFC/pages/13434886/[ACCEPTED]+RFC%3A+Bridges